app/src/main/java/org/mozilla/fenix/perf/ApplicationInitTimeContainer.kt

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.perf
+
+import android.os.SystemClock
+
+/**
+ *  A class to store the application initialization time.
+ *  Time is stores in elapsed real time nano seconds
+ */
+internal class ApplicationInitTimeContainer(
```

Comment:
nitpick (non-blocking): as far as I can tell, the git history/blame for this file was lost. To rename this file, did you use `git mv <old-path> <new-path>` (or an equivalent in the IDE)? If not, in the future, you should use the command for file renames as it preserves the file's history. If so, then it could be that the file was changed so much that git decided to create a new entry for it rather than keep the history

The git commands to fix it now are time consuming and tedious and the file is small so I don't think it's worth our time to address before landing: I just wanted to raise this for the future. :)

Reply:
Will take note of it. I was not aware of it at all